reference/mysql-compatibility.md

```diff
@@ -68,6 +68,29 @@ The operations are executed as follows:
 
 Also, starting from TiDB 2.1.18 and 3.0.4, TiDB supports using the system variable `tidb_allow_remove_auto_inc` to control whether the `AUTO_INCREMENT` property of a column is allowed to be removed by executing  `ALTER TABLE MODIFY` or `ALTER TABLE CHANGE` statements. It is not allowed by default.
 
+> **Note:**
+>
+> If the primary key is not specified, `_tibd_rowid` will be used to identify the row in TiDB. Allocation of this variable and auto-increment column will the same allocator. If the auto-increment column is specified as the primary key, then the column will be used to identify rows. So, there will be the following situation.
```

Comment:
```suggestion
> If the primary key is not specified, TiDB uses the `_tibd_rowid` column to identify rows. The values of the `_tibd_rowid` column and the auto-increment column (if there is) are assigned by the same allocator. If the auto-increment column is specified as the primary key, then TiDB uses this column to identify rows. Therefore, there might be the following situations.
```

Reply:
> 

LGTM

Reply:
My suggestion is small so I'll apply it now to allow this PR to be merged. Thanks for your contribution! @Hangzhi